scripts/gendeclarations.lua

```diff
@@ -87,6 +87,15 @@ local spaces = any_amount(branch(
       any_character
     )),
     lit('\n')
+  ),
+  -- Linemarker inserted by preprocessor
+  concat(
+    lit('# '),
```

Comment:
@oni-link Is there not a risk that this could make a preprocessor directive like `# define`?